src/pages/signin/LoginForm/BaseLoginForm.js

```diff
@@ -159,7 +159,7 @@ function LoginForm(props) {
     useEffect(() => {
         // Just call clearAccountMessages on the login page (home route), because when the user is in the transition route and not yet authenticated,
         // this component will also be mounted, resetting account.isLoading will cause the app to briefly display the session expiration page.
-        if (props.isFocused) {
+        if (props.isFocused && props.isVisible) {
```

Comment:
Sorry, why this change?

Reply:
This is so that if we get an error back in the URL after the user tries to sign in with SSO that we display the error message instead of clearing it. 

You can test this by navigating to http://localhost:8082/transition?error=meepmeep&exitTo=%2F

Reply:
Clarification - the error is supposed to be displayed on ChooseSSOOrMagicCode or ValidateCodeForm page, right?
Simply navigating to the URL does not display an error on the BaseLoginForm.

Reply:
Ah, yes - I was thinking on the ChooseSSOOrMagicCode view, though then it wouldn't show for SAML required users 🤔 

I think that can be handled more fully in a follow up PR though, I'd like to get this merged ASAP so it's easier to develop for the remaining platforms 🙏

Reply:
Sounds good 👍